src/IO/InputStreamFromROOTFile.cxx

```diff
@@ -77,8 +77,8 @@ InputStreamFromROOTFile::post_processing()
     stream_ptr->SetBranchAddress("time1", &time1);
     stream_ptr->SetBranchAddress("time2", &time2);
 
-    stream_ptr->SetBranchAddress("eventID1",&eventID1);
```

Comment:
not sure why the rename but I leave this to you to decide. just make sure no conflict with the TOF branch

Reply:
For consistency.